include/mbgl/util/position.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <mbgl/style/types.hpp>
```

Comment:
Trim the includes here to just what's needed -- looks like `constants.hpp` and `array`.